SocialCareCaseViewerApi.Tests/V1/UseCase/CreateAllocationUseCaseTests.cs

```diff
@@ -0,0 +1,39 @@
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.UseCase;
+
+namespace SocialCareCaseViewerApi.Tests.V1.UseCase
+{
+    public class CreateAllocationUseCaseTests
+    {
+        private Mock<IDatabaseGateway> _mockDatabaseGateway;
+        private CreateAllocationUseCase _classUnderTest;
+
+        private Fixture _fixture = new Fixture();
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockDatabaseGateway = new Mock<IDatabaseGateway>();
+            _classUnderTest = new CreateAllocationUseCase(_mockDatabaseGateway.Object);
+        }
+
+        [Test]
+        public void ExecuteReturnsTheRequest()
+        {
+            var stubbedRequest = _fixture.Create<CreateAllocationRequest>();
+
+            _mockDatabaseGateway.Setup(x => x.CreateAllocation(It.IsAny<CreateAllocationRequest>()))
```

Comment:
This setup is returning the request object as is. Use case should normally return an object that is relevant to the result created by the action.

Reply:
This will be resolved when the return object is defined